lib/Registrator.js

```diff
@@ -127,7 +129,9 @@ ${this._contact};expires=${this._expires}${this._extraContactParams}`);
       },
       onReceiveResponse : (response) =>
       {
-        let contact, expires, contacts = response.getHeaders('contact').length;
+        let contact;
+        let expires;
+        let numContacts = response.getHeaders('contact').length;
```

Comment:
Why the change of the name to `numContacts`?

Reply:
hehe, silly question.., I missed the `length` keyword.

Reply:
yep, at first sight I thought `contacts` was an array.